taylorjohnson141/rancid-tomatillos
# Api Calls and Movie test cd

## Types of changes 

## Checklist:  

## What’s this PR do?
- Creates apiCalls.js
- Takes all api calling methods from components of app and its children, to keep api calling functionality in it's own file
- Imports apiCalls into App, Movies, and Movie
- Creates Test for movie.js, using async/await

## Where should the reviewer start?
- Review files changed

## How should this be manually tested?
- run ```npm start``` to check that all buttons and intended browser transitions are occuring
- run ```npm test``` to ensure that Movie.test.js is passing

## Any background context you want to 
provide?
- Needed to create file to store api calling functionality
- Needed to test Movie.js

## What are the relevant tickets?
#38 for testing Movie

## Screenshots (if appropriate)

## Comments/Questions:
